packages/playground/src/weblets/tf_openwebui.vue

```diff
@@ -87,6 +68,11 @@
           }"
           require-domain
           v-model="selectionDetails"
+          v-model:rentedByMe="rentedByMe"
+          v-model:dedicated="dedicated"
+          v-model:certified="certified"
+          v-model:hasGPU="hasGPU"
+          :showGPU="true"
```

Comment:
same here, do we need this?

Reply:
It's only full VM and open web ui solutions have the GPU switch

Reply:
please use the types here https://github.com/threefoldtech/tfgrid-sdk-ts/blob/53b59af5407c2a3763e3e2aeec7886cf71763d6a/packages/playground/src/types/index.ts#L184 to check if showgpu should be true instead of passing a static true